lib/contentstack/region.rb

```diff
@@ -2,5 +2,10 @@ module Contentstack
   class Region
     EU='eu'
     US='us'
+    AZURE_NA='azure-na'
+    AZURE_EU='azure-eu'
+    PROTOCOL='https'
```

Comment:
I think PROTOCOL = 'https://', In this case you do not have to string concate :// while creating endpoint

Reply:
@sunil-lakshman PROTOCOL should no be in Region. Region is basically DB region.